sys/net/gnrc/network_layer/sixlowpan/frag/rb/Kconfig

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_MODULE_GNRC_SIXLOWPAN_FRAG_RB
+    bool "Configure GNRC 6LoWPAN Reassembly buffer"
+    depends on MODULE_GNRC_SIXLOWPAN_FRAG_RB
+    help
+        Configure GNRC 6LoWPAN Reassembly buffer using Kconfig.
+
+if KCONFIG_MODULE_GNRC_SIXLOWPAN_FRAG_RB
+
+config GNRC_SIXLOWPAN_FRAG_RBUF_SIZE
+    int "Size of the reassembly buffer"
+    default 4
+
+config GNRC_SIXLOWPAN_FRAG_RBUF_TIMEOUT_US
+    int "Timeout for reassembly buffer entries in microseconds"
+    default 3000000
+
+config GNRC_SIXLOWPAN_FRAG_RBUF_DO_NOT_OVERRIDE
+    bool "Keep all but oldest fragment when reassembly buffer is full"
+    help
+        When not set, it will cause the reassembly buffer to
+        override the oldest entry no matter what. When set, only the oldest
+        entry that is older than @ref CONFIG_GNRC_SIXLOWPAN_FRAG_RBUF_TIMEOUT_US
```

Comment:
Does Kconfig also parse this `@ref`?

Reply:
No, right now Kconfig files are not parsed for documentation. This is only kept so we don't lose track of references when documentation is generated from them in the future.

Reply:
AFAIK it does not. But as we plan to parse the doxygen documentation from Kconfig files in the long run, we decided to keep these references in.

Reply:
Ok